Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -16,19 +16,20 @@ protocol PitchSpellingNode: Hashable {
 struct PitchSpeller {
 
     // FIXME: Flesh out for all tendencies
-    static let tendencyGraph: WeightedGraph<Tendency,Double> = WeightedGraph(
+    static let tendencyGraph: Graph<Tendency> = Graph(
```

Comment:
In the fullness of time, I think we should move out all of the masking-related stuff to an extension, but no rush.